xbmc/windows/GUIMediaWindow.cpp

```diff
@@ -1532,26 +1528,21 @@ bool CGUIMediaWindow::OnPlayAndQueueMedia(const CFileItemPtr& item, const std::s
     // Remove ZIP, RAR files and folders
     CFileItemList playlist;
     playlist.Copy(*m_vecItems, true);
+
```

Comment:
Nitpick, but this is not needed. ;-)

Reply:
If you're sure that changing m_vecItems doesn't matter. The original code just looped through m_vecItems and added to PlayListPlayer one at a time. I was erring on the side of caution by making a copy.

Reply:
I was requesting that you remove the blank line you have introduced here, no code change.

Reply:
Done